.golangci.yaml

```diff
@@ -0,0 +1,6 @@
+run:
+  deadline: 7m
+
+linters:
+  disable:
+  - unused
```

Comment:
why is this disabled?

Reply:
With `unused` linter enabled, in a local container with `4.8GiB` memory limit at some point of `golangci-lint` execution it gets all memory used (at least this is output from `docker stats`) and the `golangci-lint` process gets killed with exit code 137.
With `unused` linter disabled, the memory usage is much better and the execution is ok.

Reply:
Does this mean we won't find any unused variables anymore?

Reply:
Variable which is declared but not used should be compilation error. 
The linter doc
```
unused (megacheck): Checks Go code for unused constants, variables, functions and types [fast: false, auto-fix: false]
```